### DISCUSSION TITLE
Automated sensitive species locality blocker

### DISCUSSION THREAD

**Discussion by soteropoulosd2** [ArctosDB/arctos#4884] [General]
During the Arctos working group meeting at SPNHC Thursday, May 30, we discussed the process to block specific locality information for sensitive species. Currently, users must block each individual record, which is not is not efficient; moreover, specimen holders outside sensitive areas may not know to block specific localities of species from a different state. @Jegelewicz and @campmlc requested that I submit a request for updated functionality in Arctos.

In SERNEC (Symbiota), each state submits a list of sensitive species to automatically block specific locality information for taxa of concern. I discovered this list only blocked state-sensitive species for collections held within the state of concern, and after I notified them, Symbiota added restrictions to hide specific locality information for all collections with state-sensitive species status.

We should implement similar functionality in Arctos to protect the locations of taxa of concern. Lists of state-sensitive species from natural heritage programs can be used to determine which species to block from which states to prevent poachers from pirating our database. This list should be able to be updated with changes in species of concern, automatically blocking specific localities with additions to the list and showing specific locality information of species removed from the list.

**Comment by dustymc** [comment]
Legal data regarding taxa can be uploaded (or scraped via services if they're available) to the Arctos Legal source.

I would think there would be a lot of details that would need considered for each specimen - eg, you might want to restrict collection events of wild animals but not encounters with captives, or include/exclude specimens collected before/after some date, or include/exclude various "synonyms," or .... We can certainly talk about some way of automating encumbrances, I just think it would need to be much more targeted than "anything that somehow touches a taxon with legal information."

It has always been possible to apply Encumbrances (and do most everything else) in batches; Arctos has never been limited to encumbering specimens from individual records.

**Comment by campmlc** [comment]
This is a request to create automated encumbrances, so that we can catch data before they are uploaded to  aggregators.  This relates to recent conversations we have had with Natural Heritage in NM.  @soteropoulosd2  What would be the minimum data to encumber, and for how long? specific and verbatim locality only?

**Comment by Jegelewicz** [comment]
Suggest we talk with Symbiota peeps about what rules they use.

**Comment by dustymc** [comment]
I think it's in SERNEC not Symbiota??

**Comment by soteropoulosd2** [comment]
SERNEC is part of the Symbiota consortium.

**Comment by campmlc** [comment]
Dianna - do you have a SERNEC contact?

**Comment by soteropoulosd2** [comment]
My primary contact at SERNEC is Herrick Brown <herrick.brown@gmail.com>, though I'm sure Michael Denslow <michael.denslow@gmail.com> would also be able to help.

**Comment by dustymc** [comment]
Tentatively tabling; I don't see a technical solution.

**Comment by Jegelewicz** [comment]
Would using the Arctos Legal source provide a path? Or something similar?

**Comment by dustymc** [comment]
> Arctos Legal source provide a path?

Path (by which users can apply encumbrances or etc.) - yes.

Magic - [no.](https://github.com/ArctosDB/arctos/issues/2106#issuecomment-498447302)

**Comment by Jegelewicz** [comment]
I found out how Symbiota does this in the [webinar](https://youtu.be/ipSIXOHud8A?t=1790) yesterday.

Every name in their taxonomic thesaurus can be tagged to hide locality information with a locality security status. This seems like a cool way to do this except in the case of a homonym where one needs hidden and one doesn't (although that may be very rare?).

**Comment by dustymc** [comment]
Yea, that general idea has been around forever, I'm not a fan. The black-footed ferret running around the parking lot may need to be masked; the black-footed ferret in the exhibit on the other side of the fence does not need to be masked. Whatever the taxa and situation, it always seems to turn out to be something more complicated than "let's pretend we don't know where any of these come from."

Homonyms are another way things can get complicated.

As above, "here's a report, maybe you want to make some encumbrances" seems much more defensible - stripping out the homonyms and captive tigers and such should be trivial for someone familiar with the collection, and doesn't involve Arctos acting as a Curator.

**Reply by Jegelewicz** [reply]
> here's a report, maybe you want to make some encumbrances

Sometimes I think you are living in an alternate universe! When has this ever been effective? How many notifications go unheeded on a daily basis? It isn't going to happen - nobody has time to go and figure out if the one ferret found in a parking lot should not have locality hidden and so all of the ferret localities that should be hidden will not be. I think hiding a few captive tiger localities is less evil than publishing data on all the wild ones.

**Reply by dustymc** [reply]
User: here's some data, do your thing
Arctos: https://www.youtube.com/watch?v=7qnd-hdmgfk
